### PR TITLE
IGNITE-21977 Sql. Extend test coverage for SQL feature F661(Simple tables)

### PR DESCRIPTION
**Pull request by zstan**
Thank you for submitting the pull request.

To streamline the review process of the patch and ensure better code quality
we ask both an author and a reviewer to verify the following:

### The Review Checklist
- [ ] **Formal criteria:** TC status, codestyle, mandatory documentation. Also make sure to complete the following:  
\- There is a single JIRA ticket related to the pull request.  
\- The web-link to the pull request is attached to the JIRA ticket.  
\- The JIRA ticket has the Patch Available state.  
\- The description of the JIRA ticket explains WHAT was made, WHY and HOW.  
\- The pull request title is treated as the final commit message. The following pattern must be used: IGNITE-XXXX Change summary where XXXX - number of JIRA issue.
- [ ] **Design:** new code conforms with the design principles of the components it is added to.
- [ ] **Patch quality:** patch cannot be split into smaller pieces, its size must be reasonable.
- [ ] **Code quality:** code is clean and readable, necessary developer documentation is added if needed.
- [ ] **Tests code quality:** test set covers positive/negative scenarios, happy/edge cases. Tests are effective in terms of execution time and resources.

### Notes
- [Apache Ignite Coding Guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Java+Code+Style+Guide)

### CODE REVIEW

File: modules/sql-engine/src/integrationTest/sql/basic_queries/test_simple_table.test [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# name: test/sql/basic_queries/test_simple_table.test
+# description:  F661 Simple tables
+# group: [basic_queries]
+
+query II
+SELECT * FROM (VALUES ROW('a', 1));
```
Comment (xtern) [inline modules/sql-engine/src/integrationTest/sql/basic_queries/test_simple_table.test:6]: According to the spec, Is it necessary to use the ROW function?

Comment (zstan) [reply modules/sql-engine/src/integrationTest/sql/basic_queries/test_simple_table.test:6]: you are right, fix tests a bit

File: modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/sqllogic/ItSqlLogicTest.java [hunk 2]
```diff
@@ -139,7 +139,7 @@
 @Tag(value = "sqllogic")
 @ExtendWith(SystemPropertiesExtension.class)
 @WithSystemProperty(key = "IMPLICIT_PK_ENABLED", value = "true")
-@SqlLogicTestEnvironment(scriptsRoot = "src/integrationTest/sql")
+@SqlLogicTestEnvironment(scriptsRoot = "src/integrationTest/sql", regex = "test_simple_table.test")
```
Comment (xtern) [inline modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/sqllogic/ItSqlLogicTest.java:142]:
```suggestion
@SqlLogicTestEnvironment(scriptsRoot = "src/integrationTest/sql")
```

Comment (zstan) [reply modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/sqllogic/ItSqlLogicTest.java]: done

**Review by xtern — APPROVED** [review]

**Pull request merged**